torch/utils/_python_dispatch.py

```diff
@@ -527,15 +528,10 @@ class SchemaInfo:
     outs: list[AliasInfo]
 
 
-# Can't import torch._ops.OpOverload due to circular reference
-parsed_schema_map: dict[Any, SchemaInfo] = {}
-
-
 # Given an OpOverload, returns schema information on it.
 # This is cached for efficiency, since it can involve running torchgen
+@functools.cache
```

Comment:
for my education- is this purely a "its more idiomatic this way" thing or is this faster too?